src/Utils/QtiMarshallerUtil.php

```diff
@@ -56,9 +57,23 @@ public static function marshallCollection(QtiComponentCollection $collection)
     {
         $results = [];
         foreach ($collection as $component) {
-            $results[] = self::marshall($component);
+            $html = '';
+            $class = new \ReflectionClass(get_class($component));
+            if (property_exists($component, 'data')) {
+                $property = $class->getProperty('data');
+                $property->setAccessible(true);
+                $learnosityServiceObject = ConvertToLearnosityService::getInstance();
+                $inputPath = $learnosityServiceObject->getInputpath();
+                $file = $inputPath . '/' . $property->getValue($component);
+                if (file_exists($file)) {
+                    $html = HtmlExtractorUtil::getHtmlData(realpath($file));
+                } else {
+                    echo 'File not found: ' . $file . "\n";
```

Comment:
Can use `PHP_EOL` instead of `"\n"` for better compatibility.

Reply:
Done